news/events.qmd

```diff
@@ -7,18 +7,52 @@ format:
     css: events.css   
 ---
 
-## Upcoming Events
+  ## Upcoming Events
```

Comment:
Delete space at beginning of line, does not show up as header

Reply:
![image](https://github.com/user-attachments/assets/c2cd4e50-7953-4a05-83bb-696e0c5b8695)